Demo/EarlGreyExample/README.md

```diff
@@ -11,11 +11,11 @@ different functionalities provide by EarlGrey, written in Objective-C and Swift.
 
 ### CocoaPods
 
-Execute `gem install earlgrey` then `pod install --repo-update` from the directory where you have downloaded
+Execute `gem install earlgrey` then `pod install` from the directory where you have downloaded
```

Comment:
Combine this with other md file changes in a separate PR?

Reply:
Ditto